README.md

```diff
@@ -358,6 +394,7 @@ The **settings.txt** file contains the particular settings used to run `staramr`
 * __resfinder_db_url__, __pointfinder_db_url__, __plasmidfinder_db_url__: The URL to the git repository for the ResFinder, PointFinder, and PlasmidFinder databases.
 * __resfinder_db_commit__, __pointfinder_db_commit__, __plasmidfinder_db_commit__: The git commit ids for the ResFinder, PointFinder, and PlasmidFinder databases.
 * __resfinder_db_date__, __pointfinder_db_date__, __plasmidfinder_db_date__: The date of the git commits of the ResFinder, PointFinder, and PlasmidFinder databases.
+* __mlst_version__: The version of `MLST`.
```

Comment:
Thank you for adding in the MLST version 😄

Reply:
No problem